5. Linked List/remove_impostors/RemoveImpostersMerlin.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * RemoveImpostersMerlin.java
+ * Remove y elements after the 1st x elements in a linked list
+ *
+ * Description-
+ * Let us first iterate through the the 1st x nodes of the linked list and add them to the answer
+ * Then we can skip the next y nodes
+ * Finally we add the remaining nodes to the tail of the xth node, and we have our updated list
+ *
+ * Time Complexity-O(n), Space Complexity-O(1)
+ *
+ * @author merlin[https://github.com/m-e-r-l-i-n]
+ */
+
+import java.io.*;
+import java.util.*;
+
+class Node
+{
+    int v;
+    Node next;
+    Node(int x)
+    {
+        v=x;
+        next=null;
+    }
+}
+
+class RemoveImpostersMerlin
+{
+    public static void main(String args[])throws Exception
+    {
+        BufferedReader bu=new BufferedReader(new InputStreamReader(System.in));
+        StringBuilder sb=new StringBuilder();
+        int t=Integer.parseInt(bu.readLine());
+        while(t-->0)
+        {
+            int n=Integer.parseInt(bu.readLine());
+            int i;
+            Node head=null,ptr=null;
+            String s[]=bu.readLine().split(" ");
+            for(i=0;i<n;i++)
+            {
+                int x=Integer.parseInt(s[i]);
+                Node cur=new Node(x);
+                if(head==null)
+                {
+                    head=cur;
+                    ptr=head;
+                }
+                else
+                {
+                    ptr.next=cur;
+                    ptr=ptr.next;
+                }
+            }
+
+            s=bu.readLine().split(" ");
+            int x=Integer.parseInt(s[0]),y=Integer.parseInt(s[1]);
+            removeImposters(head,x,y);
+            while(head!=null)
+            {
+                sb.append(head.v+" ");
+                head=head.next;
+            }
+            sb.append("\n");
+        }
+        System.out.print(sb);
+    }
+
+    static void removeImposters(Node head,int x,int y)
```

Comment:
The problem description is updated. Please change your solution accordingly. Sorry for the inconvenience.

Reply:
no worries. i am working on it